### PR TITLE
refactor huber_loss/argsor unittests for kunlun, *test=kunlun

### PR DESCRIPTION
**Pull request by tangzhiyi11**
### PR types
Function optimization

### PR changes
OPs

### Describe
refactor huber_loss/argsor unittests for kunlun
*test=kunlun

### CODE REVIEW

File: python/paddle/fluid/tests/unittests/xpu/test_argsort_op_xpu.py [hunk 1]
```diff
@@ -18,220 +18,81 @@
...
-        self.get_output()
-        self.outputs = {"Out": self.sorted_x, "Indices": self.indices}
-
-    def init_axis(self):
-        self.axis = 0
-
-    def init_dtype(self):
-        self.dtype = 'int'
-
+class XPUTestArgsortOp1(XPUOpTestWrapper):
```
Comment (XiaociZhang) [inline python/paddle/fluid/tests/unittests/xpu/test_argsort_op_xpu.py]: 这里为啥名字是Op1

**Comment by ykkk2333** [comment]
LGTM

File: python/paddle/fluid/tests/unittests/xpu/test_argsort_op_xpu.py (hunk 1 above)
Comment (tangzhiyi11) [reply python/paddle/fluid/tests/unittests/xpu/test_argsort_op_xpu.py]: 已修改

File: python/paddle/fluid/tests/unittests/xpu/test_argsort_op_xpu.py [hunk 2]
```diff
@@ -18,220 +18,80 @@
...
+                    self.x, kind='heapsort', axis=self.axis)
+                self.sorted_x = np.sort(self.x, kind='heapsort', axis=self.axis)
+
+        def set_xpu(self):
+            self.__class__.use_xpu = True
+            self.__class__.no_need_check_grad = True
+
+        def test_check_output(self):
+            self.check_output_with_place(self.place)
+            print('axis:', self.axis, 'descending:', self.descending)
```
Comment (XiaociZhang) [inline python/paddle/fluid/tests/unittests/xpu/test_argsort_op_xpu.py]: 这个print是否必须？

Comment (tangzhiyi11) [reply python/paddle/fluid/tests/unittests/xpu/test_argsort_op_xpu.py]: 已去掉

**Review by XiaociZhang — APPROVED** [review]
LGTM

**Review by QingshuChen — APPROVED** [review]

**Pull request merged**